CHANGES.md

```diff
@@ -1,3 +1,12 @@
+# UNRELEASED
+- [BREAKING CHANGE] Nano 7 accepted a callback to the `*AsStream` functions, but
```

Comment:
```
Users of the `*AsStream` functions should ensure they are using event handlers not callbacks before moving to this version.
```
Should we also add this as a Note to the README?

Reply:
Good point and I'll need to add something to the migration guide as well for 3 to 4.

Reply:
FYI @emlaver @smithsz - I updated the documentation in e125032 please review.

Reply:
@ricellis +1 for the doc updates